### DISCUSSION TITLE
How to add color to different agents in simulation

### DISCUSSION THREAD

**Discussion by queenxy** [google/brax#335] [Q&A]
I am doing a multi-agent task using brax, but I can't distinguish them because they have the same shape. Is there any way to add different color or patterns to the surface?

**Comment by btaba** [comment]
Hi @queenxy, yes you can add color via rgba in the XML, see the PR https://github.com/google/brax/pull/295 and example XML:
https://github.com/google/brax/blob/main/brax/test_data/colour_objects.xml
